sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestVariousOddities.java

```diff
@@ -121,4 +122,30 @@ public void testNullQueryReturn() {
      * This interface should not be loaded by any test other than {@link TestVariousOddities#testConcurrentHashCode()}.
      */
     public interface SpiffyConcurrent extends SqlObject {}
+
+    @Test
+    public void testInterfaceMIAbiguity() {
+        assertThatThrownBy(() ->
+                dbRule.getSharedHandle().attach(SpiffyParallelUniverse.class))
+            .isInstanceOf(UnableToCreateSqlObjectException.class)
+            .hasMessageContaining("SpiffyParallelUniverse has ambiguous methods");
+    }
+
+    @Test
+    public void testInterfaceMIResolution() {
+        dbRule.getSharedHandle().attach(SpiffyUniverseSaved.class);
+    }
+
+    public interface SpiffyWithMustache {
+        @SqlQuery("select id, name from something where id = :id")
```

Comment:
I think we can simplify the queries to just `select 1` and `select 2` or something instead of reusing this crazy fantasy domain, and to stick closer to the given example code (`delete from a` and `delete from b`), which made a more succinctly understandable conflict